packages/app-builder-backend/src/eject/templates/format.ts

```diff
@@ -9,6 +8,7 @@ export const slugToCamel = (str) =>
 const capitalize = (s) => s && s[0].toUpperCase() + s.slice(1)
 
 export const lint = async (code: string): Promise<string> => {
+  const baseEslint = require('@reapit/ts-scripts/src/eslint/base-eslint') // inline import because ts-scripts has dependency issues
```

Comment:
Think this is just because you need to have the eslint executable in scope in your package - can explain in the morning!